x/incentives/keeper/gauge.go

```diff
@@ -306,6 +306,15 @@ func (k Keeper) Distribute(ctx sdk.Context, gauge types.Gauge) (sdk.Coins, error
 		if err := k.bk.SendCoinsFromModuleToAccount(ctx, types.ModuleName, owner, distrCoins); err != nil {
 			return nil, err
 		}
+		lockOwner, err := sdk.AccAddressFromBech32(lock.Owner)
```

Comment:
Above lines still need to be updated for auto-staking right? (Multiplying by `1 - auto-staking` decimal above, and `auto-staking` fraction below

Reply:
The process is coins are sent to user fully by 100% and after that specific percentage is auto-staked.
The reason is because, not only OSMO is rewarded but other tokens are also rewarded as well. And it's convenient to reward everything and just delegate percentage of OSMO income for auto-staking.